setup.py

```diff
@@ -31,6 +31,6 @@
 
     packages=['otr'],
     provides=['otr'],
-    install_requires=['gmpy2', 'zope.interface', 'application', 'cryptography']
+    install_requires=['gmpy2', 'zope.interface', 'python3-application', 'cryptography']
```

Comment:
Given https://github.com/AGProjects/python-application/blob/7ce8696b94e103872f399ad0508b5601f8414aa9/setup.py#L16 and https://github.com/AGProjects/python-application/blob/7ce8696b94e103872f399ad0508b5601f8414aa9/application/__info__.py#L4 should this be:

```suggestion
    install_requires=['gmpy2', 'zope.interface', 'python-application', 'cryptography']
```

...instead?

Reply:
This is the python3 version of python-otr. It should not depend on python-application

Reply:
So python3-application is a different package than python-application but does the same thing?

Reply:
> So python3-application is a different package than python-application but does the same thing?

python3-application is the python3 version of python-application